Created a record set uploader

eedrummer opened the pull request:
This is a command line utility that will create a new record set and upload patient resources with a tag to associate them with the record set.

eedrummer committed:
Created a record set uploader

mlos100 commented:
2 suggestions (you can obviously veto)
1. Change the help message for fhir URL to mention the patient match server rather than a fhir server since you are posting to /RecordSet.  Otherwise, someone may try to use this useful utility on a regular fhir server to upload patients.
2. The application returns without error (but nothing is uploaded) if I have a trailing slash (e.g., http://localhost:3001/) in the fhirURL parameter.  Consider checking and stripping it.  The ptmach server is returning a 404 for http://localhost;3001//Patient.  (This also makes me wonderl what the proper behavior at the ptmatch server should be)

eedrummer committed:
Stripping trailing slash from fhirURL

Also better documentation on the fhirURL parameter
Better error checking and exiting on unexpected server behavior

eedrummer commented:
@mlos100 I think the latest commit addresses your comments.

mlos100 commented:
Thanks for using readable names like trimmedFhirUrl rather than a go-like tfu or pu for patientUrl
